src/Microsoft.Dnx.Host.CoreClr/DomainManager.cs

```diff
@@ -11,8 +12,24 @@
 [SecurityCritical]
 sealed class DomainManager
 {
+    [StructLayout(LayoutKind.Sequential)]
+    private unsafe struct NativeBootstrapperContext
+    {
+        public char* OperatingSystem;
```

Comment:
No. This is up to the managed code to decide what to do with this. Native code is supposed to bootstrap the runtime and give all the required data to the managed code without processing it. In other words it should not know concepts (like Runtime ID which are managed stack concepts)